components/automate-ui/src/app/pages/+compliance/+reporting/+reporting-overview/overview-trend/overview-trend.component.ts

```diff
@@ -153,7 +153,7 @@ export class OverviewTrendComponent implements OnChanges, OnDestroy  {
       .attr('x', this.vbWidth / 2)
       .attr('y', this.vbHeight - 3)
       .text(() => {
-        const format = '%B %e, %Y';
+        const format = '%a, %e %b %Y';
```

Comment:
This is a D3 time format.  Should this formatting stay here on its own or do you think I should pull this over to the datetime formatting helper?